ethcore/src/engines/authority_round/mod.rs

```diff
@@ -906,6 +971,12 @@ impl AuthorityRound {
 	fn address(&self) -> Option<Address> {
 		self.signer.read().as_ref().map(|s| s.address() )
 	}
+
+	/// AuRa step getter for testing purposes.
+	#[allow(unused)]
```

Comment:
Is it actually used in tests? Why not `#[cfg(test)]`?

Reply:
Right, it's dead code and will be removed.